cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -80,7 +109,9 @@ def import_opinions_from_recap(court=None, total_count=0):
                 # Ex. 42\u2009U.S.C.\u2009§\u200912131 \u2009 is a small space
                 # fallback to regular citation match
                 logger.warning(
-                    f"Hyperscan failed for {recap_document}, trying w/o tokenizer"
+                    "Hyperscan failed for {}, trying w/o tokenizer".format(
+                        recap_document
+                    )
```

Comment:
I think this has the same problem as before, in that it runs the .format() command when the code is run even if we're not outputting at the warning level, say. I believe the right way to do this is: 

```suggestion
                    "Hyperscan failed for %s, trying w/o tokenizer", recap_document)
```

Reply:
thanks